src/main/java/com/metamx/collections/bitmap/WrappedImmutableConciseBitmap.java

```diff
@@ -52,7 +53,10 @@ public ImmutableConciseSet getBitmap()
   @Override
   public boolean get(int value)
   {
-    return bitmap.get(value) > 0;
+    final ConciseSet conciseSet = new ConciseSet();
+    conciseSet.add(value);
+    final ImmutableConciseSet immutableConciseSet = ImmutableConciseSet.newImmutableFromMutable(conciseSet);
+    return ImmutableConciseSet.intersection(this.bitmap, immutableConciseSet).size() != 0;
```

Comment:
I did some benchmarking as well and agree with your results. I'm inclined to fix it at the `extendedset` layer as a performance improvement. Filed https://github.com/metamx/extendedset/issues/8 to track